### DISCUSSION TITLE
Get more api requests

### DISCUSSION THREAD

**Discussion by raffis** [community/community#63145] [Enterprise]
### Select Topic Area

Product Feedback

### Body

Is there a way to get more api requests besides company enterprise? I constantly get rate limited while having to work across multiple repositories.
I am aware that enterprise offers more requests and via an enterprise client, although to upgrade the company account to enterprise is quite expensive only because of this very use case and a few users needing to do these sort of things.
Is it possible to increase api requests for pro users or the ability to buy additional requests as a company team subscription?

**Comment by Akashh2151** [comment]
As of my last knowledge update in September 2021, GitHub's rate limits for API requests are based on the type of account you have, and upgrading to an Enterprise account generally provides higher rate limits. However, the specific offerings and pricing options might have evolved since then.

Here are a few potential options you can explore to address the issue of hitting rate limits while working across multiple repositories:

OAuth Tokens: GitHub API requests made using an OAuth token associated with your user account typically have higher rate limits compared to anonymous requests. If you're not already using OAuth tokens for your requests, consider implementing them in your workflow.

Personal Access Tokens: GitHub allows you to create personal access tokens that can be used to authenticate API requests. These tokens can have higher rate limits than anonymous requests. However, keep in mind that personal access tokens are tied to individual user accounts and are not meant for shared use among a team or organization.

GitHub Apps: If you're building integrations that require higher rate limits, you might consider creating a GitHub App. GitHub Apps can have higher rate limits than regular user or OAuth tokens. Additionally, GitHub Apps can be installed on multiple repositories, which could help with your scenario of working across multiple repositories.

Rate Limiting Strategies: Consider implementing rate-limiting strategies in your code to manage the available API requests more effectively. This might involve pacing your requests, caching responses, and prioritizing which API calls are essential for your workflow.

Contact GitHub Support: If you're consistently hitting rate limits and it's impacting your workflow significantly, you could consider reaching out to GitHub Support. They might be able to provide guidance or insights on how to optimize your API usage or discuss potential options for your specific use case.

Check for Updates: GitHub frequently updates its features, offerings, and pricing models. It's possible that there have been changes since my last update in September 2021. Checking the official GitHub documentation or contacting GitHub sales/support might provide you with the most current information on rate limits and available options.

Keep in mind that pricing and features can change, so it's always a good idea to consult GitHub's official resources or contact their support for the most accurate and up-to-date information tailored to your specific needs.
